go.mod

```diff
@@ -5,6 +5,7 @@ go 1.18
 require (
 	github.com/avelino/slugify v0.0.0-20180501145920-855f152bd774
 	github.com/clbanning/mxj v1.8.4
+	github.com/golang/mock v1.4.4
```

Comment:
To understand since I couldn't find that in the PR description: is this test case so unique compared to other tests in pREST that we need to add a dependency for it, or is it a general direction in the pREST to migrate tests to use this library?

I am not a maintainer, but I think that might make the onboarding more difficult to pREST; that's why I ask. Having a standard on how to test things is important.

Reply:
I'd say that it will be a tendency as we continue to develop the app to add more of this as it helps reducing the test complexity.

Also @vmesel please use the latest version, `v1.6.0`